src/configure/clients/devOps/azureDevOpsClient.ts

```diff
@@ -237,6 +237,25 @@ export class AzureDevOpsClient {
             });
     }
 
+    public async getOrganizationIdFromName(organizationName: string) {
+        let organization = (await this.listOrgPromise).find((org) => {
+            return org.accountName === organizationName;
+        });
+
+        if(!organizationName) {
+            await this.listOrganizations(true);
+            organization = (await this.listOrgPromise).find((org) => {
```

Comment:
why await again, the previous line should return all orgs

Reply:
listOrgPromise is of type promise, and hence we either need to do .then or await. I personally prefer await over .then.